### DISCUSSION TITLE
shifting authorship from source codes to user manual or other documentation

### DISCUSSION THREAD

**Discussion by carltape** [SPECFEM/specfem3d#1560] [General]
At the Toronto meeting, Rene (@gassmoeller) said that software development studies have shown that the presence of author names on source code makes people less inclined to contribute. (This kind of makes sense, since you're wondering whether you should change something that "belongs" to someone else.) He recommended ensuring that author contributions are identified in the documentation of the package rather than the source code. @danielpeter what's your take on this? It seems like a task that could be done by anyone, though probably some high-level judgement is needed. In some places of the code, it may be helpful to see some historical attribution, especially in cases where github does not preserve the history.

**Comment by gassmoeller** [comment]
In addition to making changes easier, it also better illustrates who "owns" the code. If you contribute to a community project and in order to make it a community project, the "community" should get a sense of ownership of the project, and appreciation for the contributions instead happens in the documentation (i.e. the authors document).
You are right the changes can be done by anyone (it is also a good starter project), but like you said someone from high up has to say: Let's do it! Because no new contributor (and not me) would feel comfortable removing author information from files without a clear go signal. I also remember we agreed, but it would be good to have this in writing in this discussion :-).
